builtin/logical/ssh/path_config_zeroaddress.go

```diff
@@ -0,0 +1,182 @@
+package ssh
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+// Structure to hold roles that are allowed to accept any IP address.
+type zeroAddressRoles struct {
+	Roles string `json:"roles" mapstructure:"roles"`
```

Comment:
I would make this a []string. Then you can do the comma parsing once, and use simple slice operations everywhere else.